config.schema.yaml

```diff
@@ -31,7 +31,7 @@ splunk_instance:
       type: "string"
       secret: true
       required: false
-    splunkToken:
+    splunk_token:
```

Comment:
This change means everyone that uses the pack will need to change their config to update to a new value, which is quite a breaking change for a minor update.
Is it necessary to update the name, and if it is, then it probably should go to a version 3.x as it means people will need to know they need to update their config.